### PR TITLE
K8s provider with multus cni

### PR DESCRIPTION
**Pull request by SchSeba**
Create a new k8s 1.11.0 provider with multus cni.

---
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/kubevirtci/27)

### CODE REVIEW

**Comment by rmohr** [comment]
@SchSeba does it work already?

**Comment by SchSeba** [comment]
@cynepco3hahue 
I only test it with k8s-1.11.0
Added the version variable. ( I was using an export variable locally)
Thanks!

**Comment by SchSeba** [comment]
@rmohr Sure :)

**Comment by cynepco3hahue** [comment]
@rmohr If you do not have other objections I can create the image and push it to docker hub.

File: k8s-multus/scripts/provision.sh [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+      }
+    }'
+EOF
+
+kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f clusterrole.yml
+kubectl --kubeconfig=/etc/kubernetes/admin.conf create clusterrolebinding multus-node-`hostname` \
+                                                       --clusterrole=multus-crd-overpowered \
+                                                       --user=system:node:`hostname`
+
+kubectl --kubeconfig=/etc/kubernetes/admin.conf apply -f https://raw.githubusercontent.com/intel/multus-cni/dev/network-plumbing-working-group-crd-change/examples/multus-with-flannel.yml
```
Comment (rmohr) [inline k8s-multus/scripts/provision.sh]: Can you download the file here and then just provision it from the file here and in the startup script? You don't have to repeate a mistake which we are going to fix: #23.

File: k8s-multus/scripts/node01.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+# Wait for the network to really came up
+while [[ `cat /proc/sys/net/ipv4/ip_forward` -eq 0 ]]
+do
+ sleep 2
+done
+
+while [[ ! -f /proc/sys/net/bridge/bridge-nf-call-iptables ]]
+do
+ sleep 2
```
Comment (rmohr) [inline k8s-multus/scripts/node01.sh]: @slintes looks like this may be related to the issues you saw?

Comment (rmohr) [reply k8s-multus/scripts/node01.sh]: Ok, then it might be better to wait for docker in general?

Comment (SchSeba) [reply k8s-multus/scripts/node01.sh]: Done

**Comment by SchSeba** [comment]
@rmohr I make the requested changes.

**Comment by cynepco3hahue** [comment]
Will update the PR with relevant image digest.

**Pull request merged**